cmake/DeathKnellConfig.cmake

```diff
@@ -0,0 +1,34 @@
+#.rst:
+# FindDeathKnell
+# -------
+#
+# Find libDeathKnell
+# DeathKnell deals with file access and directory traversal.
+##
+# Result variables
+# ^^^^^^^^^^^^^^^^
```

Comment:
For this and the other LROSS pull requests: 

I do not think the <repo name>Config.cmake are necessary. Further, they are hard to test also so I believe all of the special cmake config files should no be installed and in fact removed
(Murmurhash, FileIO, StopWatch, DeathKnell, Queuenado)